src/App.jsx

```diff
@@ -0,0 +1,51 @@
+import "./App.css";
+import Card from "./components/Card";
+import Board from "./components/Board";
+import x from "./assets/x.png";
+import o from "./assets/o.png";
+import { useState } from "react";
+
+function App() {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [isX, setIsX] = useState(false);
+
+  const handleSubmit = (isX = true) => {
+    setIsSubmitted(true);
+    setIsX(isX);
+  };
+
+  return (
+    <>
+      <div className="container">
+        {isSubmitted ? (
```

Comment:
Կառաջարկեի տեղերով փոխել ընտրելու քարտը, ու Board-ը, որովհետև ի սկզբանե ընտրելուննա միշտ հանդիպում օգտատիրոջը, 

Ավելի ընթեռնելի ա

Այսինքն՝ 
```suggestion
        {!isSubmitted ? (
```

Ու տեղերով փոխել